src/components/proofs/EmailProofSection.tsx

```diff
@@ -0,0 +1,30 @@
+import { AccentText, BodyText } from 'components/ui/Text'
+import { DataKeys } from 'models/DataKeys'
+import { useSnapshot } from 'valtio'
+import EmailProof from 'components/proofs/EmailProof'
+import Proof from 'models/Proof'
+import ProofSection from 'components/proofs/ProofSection'
+import ProofStore from 'stores/ProofStore'
+import ProofsList from 'components/proofs/ProofsList'
+
+export default function ({ dataKey }: { dataKey: DataKeys }) {
```

Comment:
If you'll use 
```TS
{ dataKey }: { dataKey: DataKeys }
```
Maybe we can extract this whole construction and use it like this:
```TS
export default function (dataKeyComponentProp)
```
?

Reply:
Not sure if it's possible, never did this before